### PR TITLE
Get correct tensor shape of `get_attr` op

### PR DESCRIPTION
**Pull request by pschang-phy**
### Ticket
https://github.com/tenstorrent/pytorch2.0_ttnn/issues/610

### Problem description
`get_shape` function could not properly deal with `get_attr` op

### What's changed
Get correct tensor shape of `get_attr` op.

### CODE REVIEW

**Review by swimdi — APPROVED** [review]

File: torch_ttnn/utils.py [hunk 1]
```diff
@@ -9,14 +9,19 @@ def GraphCleanup(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
...
     if isinstance(node_or_shape, (int, float)):
         return torch.Size()
     if isinstance(node_or_shape, (torch.Size, list, tuple)):
         return node_or_shape
     if isinstance(node_or_shape, torch.fx.node.Node):
         if (val := node_or_shape.meta.get("val", None)) is not None:
             return val.size()
+
+        if node_or_shape.op == "get_attr":
+            val = getattr(gm, node_or_shape.target)
```
Comment (swimdi) [inline torch_ttnn/utils.py:22]: Is it possible `val` be type of `int/float` ?

File: torch_ttnn/utils.py [hunk 2]
```diff
@@ -9,14 +9,19 @@ def GraphCleanup(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
...
+            return val.size()
```
Comment (swimdi) [inline torch_ttnn/utils.py:23]: Is it possible val be type of int/float ?

**Review by swimdi — APPROVED** [review]

File: torch_ttnn/utils.py [hunk 3]
```diff
@@ -9,14 +9,19 @@ def GraphCleanup(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
     return gm
 
 
-def get_shape(node_or_shape):
+def get_shape(gm: torch.fx.GraphModule, node_or_shape):
```
Comment (ayerofieiev-tt) [inline torch_ttnn/utils.py:12]: I think this function deserves for some doc string

**Review by ayerofieiev-tt — APPROVED** [review]

File: torch_ttnn/utils.py (hunk 2 above)
Comment (pschang-phy) [reply torch_ttnn/utils.py]: I check the doc, `getattr` could be any type. I add more check for `val` type.

File: torch_ttnn/utils.py (hunk 3 above)
Comment (pschang-phy) [reply torch_ttnn/utils.py:12]: agree. I have updated it for doc string b280d540a5e5c31bde8e75d0f312c1b2815419f8

**Pull request merged**